index.html

```diff
@@ -42,15 +54,220 @@
             </p>
             <p class="connect">LET’S CONNECT</p>
             <ul class="social-container">
-              <li><i class="fab fa-twitter"></i></li>
-              <li><i class="fab fa-linkedin-in"></i></li>
               <li><i class="fab fa-github"></i></li>
-              <li><i class="fab fa-facebook"></i></li>
-              <li><i class="fab fa-instagram"></i></li>
+              <li><i class="fab fa-linkedin-in"></i></li>
+              <li><i class="fab fa-angellist"></i></li>
+              <li><i class="fab fa-twitter"></i></li>
+              <li><i class="fab fa-medium-m"></i></li>
             </ul>
           </section>
         </section>
       </section>
+
+      <section class="projects">
+        <section class="project">
+          <section class="project-image">
+            <img
+              src="./assets/images/project1.png"
+              alt="My first project snapshot"
+            />
+          </section>
```

Comment:
- [ ] The title (e.g., "My Recent Works") uses a header tag hx, Please include the title.

Reply:
I just nested section elements here. There is no text in this part from the mockup. But I understand perfectly the point you are making. I greatly appreciate that.

Reply:
Well resolved